persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -67,7 +67,8 @@ last_notified_date = last_notified_date,
 last_read_date = last_read_date,
 mls_last_keying_material_update_date  = excluded.mls_last_keying_material_update_date,
 mls_cipher_suite = excluded.mls_cipher_suite,
-receipt_mode = excluded.receipt_mode;
+receipt_mode = excluded.receipt_mode,
+message_timer = message_timer;
```

Comment:
this means if you insert a conversation that already exists, the  message_timer will not be updated.

Reply:
I think it will because there is line ON CONFLICT(qualified_id) DO UPDATE SET

Reply:
You're right 😅